README.md

```diff
@@ -1,112 +1,141 @@
-# Location Picker Smart Widget UI (Angular)
+# NgxLocationPicker
 
-This is the Angular 6+ UI for a Smart Widget implementing a picker field to choose a location (street, address or point of interest). It is matched by a [corresponding back-end service](https://github.com/digipolisantwerp/location-picker_service_nodejs) which is needed when running it in remote mode. A default implementation for selecting locations in antwerp is provided.
+Location Picker for Angular 7+. Provides easy to use interface for searching locations or addresses within the city of Antwerp.
 
-![screenshot](example.png)
+##### Desktop view
 
-There is a demo app, see below for instructions on running it.
+![screenshot](desktop-view.png)
 
-## How to use
+##### Mobile view
 
-### Installing
+![screenshot](mobile-view.png)
 
-```sh
-> npm install @acpaas-ui-widgets/ngx-location-picker
-```
+## Using the component
+
+##### Installation
 
-### Using
+First install the component from npm:
 
-A BFF service should be running (see demo app instructions below for how to start one).
+`npm install @acpaas-ui-widgets/ngx-location-picker`
 
-Import the component in your module:
+Then import the component inside your module:
 
 ```ts
-import { LocationPickerModule } from '@acpaas-ui-widgets/ngx-location-picker';
+import {NgxLocationPickerModule} from '@acpaas-ui-widgets/ngx-location-picker';
```

Comment:
This library was originally created in context of another project that integrates with the new location picker api. We later on decided that this library could be useful for other Digipolis projects as well and that this could be a new release of the existing location picker, hence the large amount of changes. I'll review all comments and provide updates to make sure everything is in line with this repo.